qualtrics-event-gw/kubernetes/package.yaml

```diff
@@ -0,0 +1,168 @@
+apiVersion: gateway.kyma-project.io/v1alpha2
```

Comment:
Please re-think if that content should be in the connector bundle only or do you really want to have it here as well.

Reply:
removed